ekncontent/Makefile.am

```diff
@@ -19,6 +23,25 @@ lib_LTLIBRARIES =
 # Compiler flags
 AM_CFLAGS = $(STRICT_CFLAGS)
 
+# Make sure to run Gtk-doc tests and build the documentation when doing
+# 'make distcheck'
+AM_DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc
+if EOS_ENABLE_COVERAGE
+AM_DISTCHECK_CONFIGURE_FLAGS += --enable-coverage --with-coverage-dir=@EOS_COVERAGE_DIR@
+endif EOS_ENABLE_COVERAGE
+
+# Make sure that 'make dist' includes documentation
+if CAN_MAKE_DIST
```

Comment:
I think gtk-doc.m4 does this automatically, we only had to add this for `--enable-gir-doc` and `--enable-js-doc` in the parent project.